outputs/alertmanager.go

```diff
@@ -36,6 +49,25 @@ func newAlertmanagerPayload(falcopayload types.FalcoPayload) []alertmanagerPaylo
 	amPayload.Annotations["info"] = falcopayload.Output
 	amPayload.Annotations["summary"] = falcopayload.Rule
 
+	// convert priority to ves alertmanager severity
+	var severity string
+	switch strings.ToLower(falcopayload.Priority) {
+	case "emergency", "alert", "critical":
+		severity = "critical"
+	case "error":
+		severity = "major"
+	default:
+		severity = "minor"
+	}
+	// unless req. priority >= (error = 5) => drop
+	if minpri, present := os.LookupEnv("ALERTMANAGER_MINIMUMPRIORITY"); present {
+		if priorityMap[strings.ToLower(falcopayload.Priority)] < priorityMap[strings.ToLower(minpri)] {
+			return []alertmanagerPayload{}
+		}
+	}
```

Comment:
This whole block is useless, the condition for using alertmanager output according to priority is there : [handlers.go#L131](https://github.com/falcosecurity/falcosidekick/blob/59de8f56e1800ef13f83ae7603ebbafec8ad3d9d/handlers.go#L131)

Reply:
Hmm. right, shall not be here. Its because I split changes into multiple PRs. IIits for this https://github.com/falcosecurity/falcosidekick/pull/79/files#diff-85bb1a9aab9079c9155f5efc422fa23fR38

I though I can handle it better, but reading ENV is far more less complicated and fast.

Let me know whether you want to move/remove?

Reply:
You can remove lines 62-67, the method `newAlertmanagerPayload` is already only called only if priority of event > min_priority in output config